src/main/resources/hudson/os/windows/Messages_es.properties

```diff
@@ -21,9 +21,9 @@
 # THE SOFTWARE.
```

Comment:
What about the actual `Messages.properties`?

Reply:
No changes were required in `Messages.properties` if it's what you mean :)